File: Dasar-dasar Pemrograman/Semester 1/Python/Rekursi/fibonaccirec.py
def fibonacci(param):
    if param == 0 or param == 1:
        return param
        
    else:
        return (fibonacci(int(param) - 1) + fibonacci(int(param) - 2))


parameter = int(input("Masukkan batas deret: "))
n = 1

print("Deret fibonacci: ")
for i in range(1, int(parameter) + 1):
    print(fibonacci(n)," ", end="")
    n += 1

File: Dasar-dasar Pemrograman/Semester 1/JavaScript/Rekursi/fibonaccirec.js
const readline = require("readline")
const rl = readline.createInterface({
    input: process.stdin,
    output: process.stdout
});

function fibonacci (param) {
    if (param == 0 || param == 1){
        return param;
    } else {
        return (fibonacci(param - 1) + fibonacci(param - 2));
    }
}

let n = 1;

rl.question("Masukkan batas deret: ", function(batas) {
    for (i = 1; i <= batas; i++) {
        console.log(fibonacci(n));
        n++;
    }
    
    rl.close();
});

rl.on("close", function() {
    process.exit(0)
});
